cumulusci/robotframework/tests/test_cumulusci_library.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+from unittest import mock
+import unittest
+from cumulusci.robotframework.CumulusCI import CumulusCI
+from cumulusci.core.config import UniversalConfig, BaseProjectConfig
+from cumulusci.core.tests.utils import MockLoggerMixin
+from cumulusci.core.exceptions import TaskNotFoundError
+from cumulusci.utils import temporary_dir
+
+
+class TestCumulusCILibrary(MockLoggerMixin, unittest.TestCase):
+    def setUp(self):
+        self.universal_config = UniversalConfig()
+        self.project_config = BaseProjectConfig(
+            self.universal_config,
+            {
+                "project": {"name": "Test"},
+                "tasks": {
+                    "get_pwd": {
+                        "class_path": "cumulusci.tasks.command.Command",
+                        "options": {
+                            "command": "pwd",
+                        },
+                    },
+                },
+                "sources": {
+                    "example": {"path": "/tmp"},
+                },
+            },
+        )
```

Comment:
This looks fine as a way of setting up the project config. We have some utils around that provide a more shorthand way to do it for some tests (search for create_project_config), but I don't that would work here since the test covers getting the task from the project_config, so it needs to be set up including a task config.